app/views/instruments/_instrument_fields.html.haml

```diff
@@ -28,7 +28,8 @@
   = f.simple_fields_for :relay, @product.relay || @product.build_relay do |rf|
     = rf.input :type, collection: options_for_relay, include_blank: false
     = rf.input :ip
-    = rf.input :port
+    = rf.input :ip_port
```

Comment:
Just a thought: do you think it would be worth adding a hint for this. Something like "leave blank unless your device is on a non-standard port"?

And maybe for `outlet`, "e.g. 1 or 2" (or something similar if you have a better suggestion).

I'm not committed to this, but thinking it may be a good idea

Reply:
yeah, I'm good with that.